airqo-platform/AirQo-api
# Bug in registration

- What does this PR do?
This PR fixes a bug in user registration - allows the user to be redirected to the Login upon successful registration.
- What JIRA task is related to this PR?
There is no bug fix card for this PR.
- How do I test out this PR?
Git clone this repo `cd AirQo-api/src/auth-service` Then run `npm install` to get all the necessary dependencies Afterwards run `npm start` to test out the end points using any REST client tool available to you.
- Which end points should are ready for testing?
[AirQo REST API documentation](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit?usp=sharing)
partial url: /api/v1/users/register
